.travis.yml

```
@@ -1,23 +1,19 @@
 language: python
 
-sudo: false
 cache: pip
 
 python:
     - "2.7"
     - "pypy"
-    - "3.4"
     - "3.5"
-    - "3.6-dev"
+    - "3.6"
 
 matrix:
     include:
-      - python: "3.7-dev"
-        dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
-        sudo: required  # required for Python 3.7 (travis-ci/travis-ci#9069)
+      - python: "3.7"
```

```suggestion
      - python: "3.7-dev"
```
That's actually on purpose to help test Python 3.7 bugfix releases before they go public.

---

Without __dist: xenial__ this would be a ___pre-release___ Python 3.7 instead of a current build Python 3.7.